policy/modules/services/modemmanager.te

```diff
@@ -19,7 +19,7 @@ allow modemmanager_t self:process { getsched setpgid setsched signal };
 allow modemmanager_t self:fifo_file rw_fifo_file_perms;
 allow modemmanager_t self:unix_stream_socket { connectto create_stream_socket_perms };
 allow modemmanager_t self:netlink_kobject_uevent_socket create_socket_perms;
-allow modemmanager_t self:netlink_route_socket { create getattr getopt nlmsg_write read write };
+allow modemmanager_t self:netlink_route_socket { create getattr getopt setopt nlmsg_write read write };
```

Comment:
I think we should probably bump this up to `{ create_socket_perms nlmsg_write }`.